README.md

```diff
@@ -188,6 +188,35 @@ If the Azure identity is in the same resource group as your AKS cluster nodes, y
 az role assignment create --role "Managed Identity Operator" --assignee <sp id> --scope <full id of the managed identity>
 ```
 
+### Disable aad-pod-identity for a specific pod/application
+
```

Comment:
Since the top level readme is growing long, my suggestion is to create a new README under the doc/readmes and then link from table of contents of top README to the new README.